device/mellanox/x86_64-mlnx_msn2700-r0/psu_sensors.json

```diff
@@ -0,0 +1,421 @@
+{
```

Comment:
as this is a file to be common for all mellanox devices, should we locate it under the mellanox directory and not under s a specific device? is it possible?

Reply:
The "mellanox" directory exclusively comprises folders, each corresponding to a specific device. While I could technically relocate this file there, I believe doing so would disrupt the file hierarchy. Therefore, I opt to retain it within one of these folders, specifically, the "x86_64-mlnx_msn2700-r0" directory.